include/connection.h

```diff
@@ -20,8 +20,9 @@
 /* maximum duration to wait for connection */
 #define CLIENT_CONNECT_TIMEOUT_SECONDS "5"
 
-/* maximum (textual) lengths of hostname */
+/* maximum (textual) lengths of hostname and port */
 #define MAX_NODE_LENGTH 255
+#define MAX_PORT_LENGTH 10
```

Comment:
That is the maximum port, but the old code accepted ten digits, I think because that's the largest integer. We're just reading the file, not validating the semantics of the values; another piece of the software will error if the port number is illegal.